lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -2017,6 +2025,25 @@ formGraphOp(SILTensorOpInfo &opInfo,
         return diagnoseInvalidAttr("may not use this device name");
     }
 
+    auto verifyShapeAttr = [&](SymbolicValue constValue) -> bool {
+      // strip-away the possible aggregate wrapper.
+      constValue = constValue.lookThroughSingleElementAggregates();
+      if (constValue.getKind() != SymbolicValue::Array) {
+        constValue.dump();
```

Comment:
Please don't include dump() calls in the final patch.

Reply:
Removed. I was confusing this with a code path for "assert" or llvm_unreechable.